Library/ViewModels/ProjectPamphletContentViewModel.swift

```diff
@@ -37,28 +37,19 @@ ProjectPamphletContentViewModelInputs, ProjectPamphletContentViewModelOutputs {
 
   //swiftlint:disable:next function_body_length
   public init() {
-    let project = Signal.combineLatest(
-      self.configDataProperty.signal.skipNil().map(first),
+    let projectAndLiveStreamEvents = Signal.combineLatest(
+      self.configDataProperty.signal.skipNil(),
       self.viewDidLoadProperty.signal
       )
       .map(first)
 
-    let liveStreamEvents = Signal.combineLatest(
-      self.configDataProperty.signal.skipNil().map(second),
-      self.viewDidLoadProperty.signal
-      )
-      .map(first)
+    let project = projectAndLiveStreamEvents.map(first)
 
     self.loadProjectAndLiveStreamsIntoDataSource = Signal.combineLatest(
-      project,
-      liveStreamEvents,
-      Signal.merge(
-        self.viewDidAppearAnimatedProperty.signal.filter(isTrue),
-        self.viewWillAppearAnimatedProperty.signal.filter(isFalse)
-        )
-        .take(first: 1)
+      projectAndLiveStreamEvents,
+      self.viewWillAppearAnimatedProperty.signal.take(first: 1)
```

Comment:
I don't know the reason we had the merge of these different appearance events, so we should chat about this @mbrandonw. I changed this to just `viewWillAppear` so that the skeleton of the project could appear while the main project cell loads. This way the subpage cell and title can show some content until the project gets refreshed instead of the blank space. Problem is, after merging with master, this is happening at lightning speed, as if the full project is being loaded much earlier than before. It's interfering with page swiping and locking up the UI. Need to look into it.